app/src/components/v-table/v-table.vue

```diff
@@ -283,9 +283,11 @@ function updateSort(newSort: Sort) {
 				</template>
 			</table-header>
 			<thead v-if="loading" class="loading-indicator" :class="{ sticky: fixedHeader }">
-				<th scope="colgroup" :style="{ gridColumn: fullColSpan }">
-					<v-progress-linear v-if="loading" indeterminate />
-				</th>
+				<tr>
+					<th scope="colgroup" :style="{ gridColumn: fullColSpan }">
```

Comment:
```suggestion
				<tr :style="{ gridColumn: fullColSpan }">
					<th scope="colgroup">
```

It's the `loading-indicator` class that has the grid, so the gridColumn should be moved up to the `tr` to make sure the styling still works